### PR TITLE
Commented code for popup video

### LINKED ISSUE #2504: Take out Popup video
Please take out the popup video in the service. For now, please comment out the code. We might need the feature later in another context.

Screenshot from 2019-07-10 08-57-00

### PR DESCRIPTION
**Pull request by samagragupta**
Fixes #2504 

Changes:  Taken out the popup video in the service

Demo Link: https://pr-2508-fossasia-susi-web-chat.surge.sh

### CODE REVIEW

**Comment by samagragupta** [comment]
@harsh-jindal Sorry for the wrong changes. 
Fixed it

**Comment by harsh-jindal** [comment]
@samagragupta Surge link not working

**Comment by samagragupta** [comment]
@mariobehling Done

File: src/App.js [hunk 1]
```diff
@@ -131,7 +131,7 @@ class App extends Component {
 
     const renderCookiePolicy =
       showCookiePolicy === true ? <CookiePolicy /> : null;
-    const renderDialog = isModalOpen || !visited ? <DialogSection /> : null;
```
Comment (fragm3) [inline src/App.js]: Comment the code, do not remove it

Comment (samagragupta) [reply src/App.js]: @fragm3 Done

**Pull request merged**